### ISSUE TITLE
multiplayer

### ISSUE THREAD

**Issue by ddorstijn** [CodeArtemis/TriggerRally#47]
Is it true you can't go multiplayer anymore. I kinda tried to prove it doesn't work but i have 1 pro and 1 fail pc where i can't see the ground, so i wasn't able to check it out well.

Also, how are things going around there. I don;t hear much from you anymore and the changelog is outdated. where are you working on at the moment and how is the progress?

**Comment by ddorstijn** [comment]
Just for you. Really nice job. if i compare(i think) original triggerrally with youre triggerrally I think that game (almost) suchs not so bad but... you know.

How is it going with modeling the stuff?

some higher quality of ground would be amazing. On the moment je can clearly see the small textures of dirt in a row on the ground.

Ow \, how is it going with the upcoming dirt :-)

**Comment by jareiko** [comment]
Thanks! Good to know that I've improved on the original.

Yes, sorry, multiplayer is turned off because I broke it while reworking some code that touches it. I wasn't aware anyone was using it! It will be back soon. Fingers crossed. ;)

Sorry that I haven't been updating the changelog. I don't really have a good workflow for that, so I forget about it. I have been pushing some small changes, but nothing major. Mostly I'm working on much-needed rewrites to enable the bigger new things. Such as a menu screen! :smiley:

One problem I'm having at the moment is that adding new features to the editor is taking more and more work each time. I need to restructure the way code is balanced between server and client, so that the server just provides raw data, and the client (particularly the editor) is more independent, acting more like a typical desktop app would. Then I can do the things that have been requested, such as a list of your tracks that you can switch between, plus other nifty things.

In short, lots to do! I will let you know about the bigger things as they're done.

Improving terrain textures: haven't started working on that.
New models: I have some leads but nothing to announce yet. :)

**Comment by copiousopioids** [comment]
Yeah, I noticed the random other cars that would pop onto Alpina. I wasn't sure it it was real-time or just a ghost car of mine or someone else's run. I noticed it disappeared with V3 as well.
